monocle/github/graphql.py

```diff
@@ -86,7 +90,10 @@ def getRateLimit(self):
         return data['data']['rateLimit']
 
     @retry(
-        before=before_log(log, logging.INFO), wait=wait_exponential(min=10, max=300),
+        after=after_log(log, logging.INFO),
+        wait=wait_exponential(),
+        stop=stop_after_attempt(3),
```

Comment:
If I understand well, this 3 failures before we will manage to reduce the bulk size in the caller ?

Reply:
yes because it seems to depend on the machine answering: sometime it's working on another attempt

Reply:
ok

Reply:
stop_after_attempt=2 is enough